### ISSUE TITLE
Cannot redefine property: 0

### ISSUE THREAD

**Issue by akwilliamson** [Polidea/react-native-ble-plx#306]
I am writing a message to an RX characteristic and I have to split it into 2 messages due to limitations of the device. The first of two messages is a 20 byte uint8 buffer, the second of two messages is a 2 byte uint8 buffer. I don't start the second message until the first is completed, hence the `await` below. However on this line:

`await ble.writeCharacteristicWithResponseForDevice(deviceId, charUUID, rxUUID, message1);`

I get the error:

```
Unknown error occurred. This is probably a bug! Check reason property.
    at new BleError (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:131859:110)
    at parseBleError (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:131889:18)
    at BleManager._callPromise$ (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:132138:51)
    at tryCatch (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:18873:19)
    at Generator.invoke [as _invoke] (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:19046:24)
    at Generator.prototype.(anonymous function) [as throw] (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:18916:23)
    at tryCatch (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:18873:19)
    at invoke (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:18949:22)
    at blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:18961:15
    at tryCallOne (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:5637:14)
    at blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:5738:17
    at blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:5095:21
    at _callTimer (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:4984:9)
    at _callImmediatesPass (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:5020:9)
    at Object.callImmediates (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:5239:14)
    at MessageQueue.__callImmediates (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:4573:18)
    at blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:4400:18
    at MessageQueue.__guardSafe (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:4557:11)
    at MessageQueue.flushedQueue (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:4399:14)
    at MessageQueue.invokeCallbackAndReturnFlushedQueue (blob:http://192.168.1.180:8081/a68c25ef-c672-4a9c-a55d-3a26575b76b1:4392:21)
    at http://192.168.1.180:8081/debugger-ui/debuggerWorker.js:70:58
```
The reason property states:
`Cannot redefine property: 0`

I've been able to write previous commands the same way before reaching this command. Is this a bug as the error suggests or could it be something else?
